Documentation/deploy-master.md

```diff
@@ -6,12 +6,9 @@ See the [CoreOS Documentation](https://coreos.com/os/docs/latest/) for guides on
 
 Manual configuration of the required master node services is explained below, but most of the configuration could also be done with cloud-config, aside from placing the TLS assets on disk. For security reasons, these secrets should not be stored in cloud-config.
 
-The instructions below configure the required master node services using two main directories:
+The instructions below configure the required master node services using:
 
-* `/etc/kubernetes/manifests` for services that the kubelet should run on every master node
-* `/srv/kubernetes/manifests` for services that should run only on a single master node at a time (determined by a leader election).  The podmaster on that node manages the startup of these services by copying their manifests from here to the `/etc/kubernetes/manifests` directory, and the kubelet picks them up from there.
-
-It is vital to understand the distinction between the two and ensure that the scheduler and controller manager manifests are not accidentally placed directly in the `/etc/kubernetes/manifests` directory.
+* `/etc/kubernetes/manifests` for services that the kubelet should run on every master node. The kubelet will pick up services placed here.
 
 If you are deploying multiple master nodes in a high-availability cluster, these instructions can be repeated for each one you wish to launch.
```

Comment:
Since we lost some of the important background on how these services are master-elected, is it logical to add that here?

> ...repeated for each one you wish to launch. Each of the master services has leader-election built in, ensuring that important cluster management is handled by the leader, with the rest of the master nodes ready to take over at any time.